examples/jsm/postprocessing/SSAARenderPass.js

```diff
@@ -95,7 +95,19 @@ class SSAARenderPass extends Pass {
 		const roundingRange = 1 / 32;
 		this.copyUniforms[ 'tDiffuse' ].value = this.sampleRenderTarget.texture;
 
-		const width = readBuffer.width, height = readBuffer.height;
+    const view = this.camera.view !== null && this.camera.view.enabled ? Object.assign( {}, this.camera.view ) : null;
```

Comment:
Do you mind fixing the formatting?

Reply:
Sure. I'm refining things a bit, as well.